src/languages/Language.php

```diff
@@ -16,15 +16,17 @@ abstract class Language
 	public const RUSSIAN = 'ru';
 	public const SPANISH = 'es';
 	public const POLISH = 'pl';
-	
-	private const SUPPORTED_LANGUAGE_CLASSES = [
+	public const ITALIAN = 'it';
+
+    private const SUPPORTED_LANGUAGE_CLASSES = [
```

Comment:
This one too :D

Reply:
I don't know why phpstorm didn't works on them using code refactor